plugins/woocommerce/src/Internal/Admin/Settings.php

```diff
@@ -233,9 +234,30 @@ public function add_component_settings( $settings ) {
 		$settings['connectNonce']                     = wp_create_nonce( 'connect' );
 		$settings['wcpay_welcome_page_connect_nonce'] = wp_create_nonce( 'wcpay-connect' );
 
+		$settings['features'] = $this->get_features();
```

Comment:
It feels like we have 2 sources of truth for features now between this and [wcAdminFeatures](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/src/Admin/Features/Features.php#L327).

I like that this new set includes more information around the feature, but curious if the plan to eventually phase the old one out?